ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java

```diff
@@ -57,256 +50,12 @@ public void initMenu(AssetManager assetManager, InputManager inputManager
      * Creates the start screen.
      */
      private void createStartScreen() {
-         nifty.addScreen("start", new ScreenBuilder("start") {{
-             controller(new com.github.migi_1.Context.screens.MainMenuFunctions());
-             //Create the background layer (completely black)
-             layer(createBGLayer());
-
-             layer(new LayerBuilder("foreground") {{
-                     childLayoutVertical();
-                     backgroundColor("#0000");
-                     //Create the "Carried away bar at the top of the screen.
-                     panel(panelWithText("panel_top", "center", "center", "none", "#f008", "25%", "75%",
-                             createDefaultText("CarriedAway")));
-
-                     //Create an info panel in the middle of the screen.
-                     panel(panelWithText("panel_mid", "center", "center", "center", "#0f08", "50%", "75%",
-                             createDefaultText("Press Start to host a server \n Press Quit to exit the game")));
-
-                     //Create a panel at the bottom of the screen in which there will be buttons to go to the join screen or to exit the game.
-                     panel(new PanelBuilder("panel_bottom") {{
-                         childLayoutHorizontal();
-                         alignCenter();
-                         backgroundColor("#00f8");
-                         height("25%");
-                         width("75%");
-                         panel(panelWithControl("panel_botton_left", "center", "none", "center", "#44f8", "50%", "50%",
-                                 createButtonControl("StartButton", "Start", "none", "center", "center", "50%", "50%", "toScreen(host)")));
-                         panel(panelWithControl("panel_botton_right", "center", "none", "center", "#88f8", "50%", "50%",
-                                 createButtonControl("QuitButton", "Quit", "none", "center", "center", "50%", "50%", "quitGame()")));
-
-                     }});
-             }});
-
-         }}.build(nifty));
+         nifty.fromXml("com/github/migi_1/Context/screens/main_screen.xml", "start", new MainMenuFunctions());
```

Comment:
I think this should be moved to src/main/resources (default maven resource folder). This applies to all .xml files.